environs/config/config.go

```diff
@@ -560,6 +567,10 @@ var defaultConfigValues = map[string]interface{}{
 	MaxStatusHistorySize: DefaultStatusHistorySize,
 	MaxActionResultsAge:  DefaultActionResultsAge,
 	MaxActionResultsSize: DefaultActionResultsSize,
+
+	// By default the Juju backend is used.
+	SecretStoreKey:       "",
```

Comment:
Would it not be better here to have a const key that says juju-store? It read's a lot better to be explicit instead of documenting implicit behavior somewhere else. If our default is this state then let's be explicit about it here.

Reply:
This is done to allow us to know if the value is unset or not. On k8s, if unset, we currently use the k8s backend. But we still allow people to choose "juju" as the backend. Defaulting this to "juju" takes away the ability to know what the user wants.